### PR TITLE
Feature/vale linter

### PR DESCRIPTION
**Pull request by jacobprall**

### CODE REVIEW

**Comment by Loquacity** [comment]
Looks like it's working nicely! 🎉

**Review by Loquacity — APPROVED** [review]

File: .github/styles/vocab.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+eg
```
Comment (Loquacity) [inline .github/styles/vocab.txt:1]: Why did you include eg?

Comment (jacobprall) [reply .github/styles/vocab.txt:1]: It was erroring out on the use of eg for exempli gratia, but we can remove it if we want to target and remove eg

Comment (Loquacity) [reply .github/styles/vocab.txt:1]: Yeah, I want "for example" everywhere.

**Pull request merged**